documentation/modules/upgrading/proc-upgrade-cruise-control-api-version.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// assembly-upgrade-resources.adoc
+
+[id='proc-upgrade-cruise-control-api-version-{context}']
+= Upgrading Cruise Control to support {ApiVersion}
+
+.Prerequisites
+
+* A Cluster Operator supporting the `{ApiVersion}` API version is up and running.
+
```

Comment:
Do you want to add a prereq about Cruise Control being configured + link?

Reply:
Perhaps we should have a prereq link for all the optional Kafka config

Reply:
Good suggestion. I added prerequisites for:
- Cruise Control
- Mirror Maker
- Mirror Maker 2.0
- Kafka Bridge
- Kafka Connectors
- Kafka Rebalance